config/config.ts

```diff
@@ -9,10 +9,10 @@ export const config = {
 	jwt: {
 		passportSecretOrKey:
 			'askdjamsjdams djf asdkfj asjdfmasdjfmoasd ajflasdjfio asdufp83[][][]4ijt9420 fmwepa fm -2t u2q490ug qu g-04t m1 q032u59 qw89yt 240q 04q0- fwjavdsmkdnt539q840ewfi odsvjiboq3rpwef9jvnw498pq39ugrjv mpwn4p gjbn',
-		salt: 'aomihdfadfhalmsdj893rh 892rymd 2q89rync2hcodfa;skncasldjkvba9s haiopj; caiodmhf892 h[fp\';alskdoqp\'wd 90wfduqjpefio ;r893gqh fnvwda; lwajscnavupo2\'-31ojwp;nOLDhIEWAVKNSDLu ;pqej2IEO QWF;J ji;LKH;OIHlkhkH D;lkh',
+		salt: "aomihdfadfhalmsdj893rh 892rymd 2q89rync2hcodfa;skncasldjkvba9s haiopj; caiodmhf892 h[fp';alskdoqp'wd 90wfduqjpefio ;r893gqh fnvwda; lwajscnavupo2'-31ojwp;nOLDhIEWAVKNSDLu ;pqej2IEO QWF;J ji;LKH;OIHlkhkH D;lkh",
 	},
 	email: {
-		email: 'megak.gr7@gmail.com',
-		pass: 'zjefjbxwshlhkxvx',
+		email: 'megak.grupa7@gmail.com',
+		pass: 'iFimSisV0y-HYdp7pjDa',
```

Comment:
a skąd zmiana hasła i maila ? tutaj nie powinno sie nic zmienić

Reply:
Bo te nie dzialalo, to zalozylem nowe.